test/functional/unified-spec-runner/match.ts

```diff
@@ -326,29 +326,31 @@ export function expectErrorCheck(
     return;
   }
 
+  const expectMessage = `\n\nOriginal Error Stack:\n${error.stack}\n\n`;
```

Comment:
When getting some matching issue with any error I think it'll be handy to print out where the error came from, was using this to debug but seems good to keep.